src/app/_components/PageSection.tsx

```diff
@@ -33,16 +34,17 @@ export function PageSection({
   image,
   children,
   cta,
+  fullWidth,
```

Comment:
I would find it more intuitive if using full width is the default - And we have a prop, like, `shouldConstrainTextWidth` set to default `true`.

In that case, we wouldn't need the double negation either.

```
      <div
        className={clsx('mb-6 mt-4', {
          'grid grid-cols-1 gap-6 lg:grid-cols-2': image,
          'max-w-readable': shouldConstrainTextWidth,
        })}
      >
```

Reply:
After a second look, we don't need any any of that!

`max-w-readable` only applies to the `Heading` and `DescriptionText, not `children`, which is outside of this container.

So, it's all good. Thanks for flagging :)